### PR TITLE
Adds ScriptureFeature to ContentItem feature

### PR DESCRIPTION
**Pull request by vinnyjth**
## DESCRIPTION
image

### What does this PR do, or why is it needed?
Hooks up ScriptureFeature component from the API to the Features component in the app. 

### What design trade-offs/decisions were made?

n/a

### How do I test this PR?

Open the "Spur one another" content item in the "Gathering" sermon. 

### What automated tests did you write?

n/a

## TODO:

- [x] PR has a relevant title that will be understandable in a public changelog (ie...non developers)
- [ ] Closes [tag-issues-here]
- [x] No new warnings in tests, in storybook, and in-app
- [x] Upload GIF(s) of iOS and Android
- [x] Set a relevant reviewer

## REVIEW:

**Manual QA**

- [ ] Manual QA on iOS and ensure it looks/behaves as expected
- [ ] Manual QA on Android and ensure it looks/behaves as expected

**Code Review: Questions to consider**

- [ ] Read through the "Files changed" tab _very carefully_
- [ ] Edge cases: what assumptions are made about input?
- [ ] What kind of tests could be written?
- [ ] How might we make this easier for someone else to understand?
- [ ] Could the code be simpler?
- [ ] Will the code be easy to modify in the future?
- [ ] What's one part of these changes that makes you excited to merge it?

> The purpose of PR Review is to _improve the quality of the software._

### CODE REVIEW

File: packages/apolloschurchapp/src/content-single/Features/index.js [hunk 1]
```diff
@@ -4,23 +4,30 @@ import { Query } from 'react-apollo';
...
+      query={GET_CONTENT_ITEM_FEATURES}
+      fetchPolicy="cache-and-network"
+      variables={{ contentId }}
+    >
       {({ data: { node } = {}, loading, error }) => {
         if (error) return <ErrorCard error={error} />;
         if (loading) return null;
 
         const features = get(node, 'features', []);
+        console.log(node, features);
```
Comment (redreceipt) [inline packages/apolloschurchapp/src/content-single/Features/index.js]:
```suggestion
```

Comment (vinnyjth) [reply packages/apolloschurchapp/src/content-single/Features/index.js]: thanks dude

File: packages/apolloschurchapp/src/content-single/Features/index.js [hunk 2]
```diff
@@ -4,18 +4,24 @@ import { Query } from 'react-apollo';
...
   return (
-    <Query query={GET_CONTENT_ITEM_FEATURES} variables={{ contentId }}>
+    <Query
+      query={GET_CONTENT_ITEM_FEATURES}
+      fetchPolicy="cache-and-network"
+      variables={{ contentId }}
+    >
       {({ data: { node } = {}, loading, error }) => {
         if (error) return <ErrorCard error={error} />;
         if (loading) return null;
```
Comment (redreceipt) [inline packages/apolloschurchapp/src/content-single/Features/index.js]: I know this is a holdover from a previous PR but you have `isLoading` set to required on `ScriptureFeature` and then we're not passing it in.

Comment (vinnyjth) [reply packages/apolloschurchapp/src/content-single/Features/index.js]: yah, I think I'm missing the `withIsLoading` HOC. don't merge this until I figure that out

Comment (redreceipt) [reply packages/apolloschurchapp/src/content-single/Features/index.js]: why can't you just drop a `loading` prop into `Feature` below?

**Pull request merged**